### ISSUE TITLE
JavadocParagraph with lines breaks

### ISSUE THREAD

**Issue by Schalli1987** [checkstyle/checkstyle#3391]
When writing a javadoc like

```
/**
 * This is a class that does anything.
 * <p>
 * This is a paragraph.<br>
 * It contains a line break.
 * </p>
 */
```

The Checkstyle rule JavadocParagraph shows two warnings:

`Javadoc Paragraph: <p> tag should be placed immediately before the first word, with no space after.`
and
`Javadoc Paragraph: <p> tag should be preceded with an empty line.`

If the `<br>` tag is removed, no warning is shown, as expected, but the javadoc looks not as I want it.

---

I expect, that in this case no warning is shown, since the paragraph tag is correctly inserted and closed.
##

**Comment by romani** [comment]
Please report issue as required, please read http://checkstyle.sourceforge.net/report_issue.html

**Comment by Schalli1987** [comment]
Sorry, I thought it was enough description... here you are:

/var/tmp $ javac Test.java

/var/tmp $ cat Test.java

```
public class Test {

    /**
     * This is a class that does anything.
     * <p>
     * This is a paragraph.<br>
     * It contains a line break.
     * </p>
     */
    public void a() {
        // This Javadoc generates two warnings.
    }

    /**
     * This is a class that does anything.
     * <p>
     * This is a paragraph.
     * It contains no line break.
     * </p>
     */
    public void b() {
        // This Javadoc doesn't generate any warnings, but it is not what I wanted.
    }

}
```

/var/tmp $ cat config.xml
```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE module PUBLIC "-//Puppy Crawl//DTD Check Configuration 1.3//EN" "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">

<module name="Checker">
    <property name="severity" value="warning"/>
    <property name="charset" value="UTF-8"/>
    <property name="fileExtensions" value="java, xml, properties"/>

    <module name="TreeWalker">
        <module name="JavadocParagraph"/>
    </module>
</module>
```

/var/tmp $ java -jar checkstyle-7.1-all.jar -c config.xml Test.java
```
Starting audit...
[WARN] /home/schaller/Desktop/checkstyle/Test.java:5: <p> tag should be placed immediately before the first word, with no space after. [JavadocParagraph]
[WARN] /home/schaller/Desktop/checkstyle/Test.java:5: <p> tag should be preceded with an empty line. [JavadocParagraph]
Audit done.
```

---------------

I expect, that in this case no warning is shown, since the paragraph tag is correctly inserted and closed.

**Comment by rnveach** [comment]
This is `br` tag inside `p` tag, which is the same as Issue #3810

**Comment by rnveach** [comment]
i am closing this in favor of the other issue
